lib/cli/index.js

```diff
@@ -5,9 +5,17 @@ var runBinary = require('./run-binary');
 // commands
 var add = require('./cmd-add');
 var help = require('./cmd-help');
+var init = require('./cmd-init');
 
 program.version(require('../../package.json').version);
 
+// donejs new
+program.command('new <folder>')
+  .description('Create and initialize DoneJS application in folder')
+  .action(function(folder, params) {
```

Comment:
For the `new` command we should also have the usage information like `add` does.